OpenRA.Game/FileSystem/Folder.cs

```diff
@@ -32,10 +32,13 @@ public IEnumerable<string> Contents
 		{
 			get
 			{
+				var contents = new List<string>();
```

Comment:
This can simplify down to:
```csharp
return Directory.GetFiles(path, "*", SearchOption.TopDirectoryOnly)
	.Concat(Directory.GetDirectories(path))
	.Select(Path.GetFileName)
	.OrderBy(f => f);
```

Reply:
LINQified it.